storage/objects/objects_test.go

```diff
@@ -189,9 +189,11 @@ func TestKMSObjects(t *testing.T) {
 
 	kmsKeyName := fmt.Sprintf("projects/%s/locations/%s/keyRings/%s/cryptoKeys/%s", tc.ProjectID, "global", keyRingID, cryptoKeyID)
 
-	if err := uploadWithKMSKey(ioutil.Discard, bucket, object, kmsKeyName); err != nil {
-		t.Errorf("uploadWithKMSKey: %v", err)
-	}
+	testutil.Retry(t, 10, time.Second, func(r *testutil.R) {
```

Comment:
Perhaps we should wait for a little longer before retrying (maybe `10*time.Second`) since this is related to quota.

Reply:
This failure is actually a permissions issue and not a rate limit issue! See #1551

Reply:
Gotcha. Thanks!